scripts/shared_env

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This script sets up the GOPATH and PATH that the agent expects, as well as
+# setting any other common environment variables
+# It is expected to be 'sourced' while the CWD is the root of the repository
+
+# Manually setup godep path; avoid build-time dependency on the godep binary
+# The existing GOPATH needs to have the agent itself in it, but shouldn't need
+# anything else
+export GOPATH="`pwd`/agent/Godeps/_workspace/:$GOPATH"
```

Comment:
Can you change this to `export GOPATH="$(pwd)/agent/Godeps/_workspace/:${GOPATH}"`?  Similar changes for the `PATH` variable below.